### PR TITLE
creating branch only for React Eventonica

### PR DESCRIPTION
**Pull request by GrunuffyGit**

### CODE REVIEW

**Comment by GrunuffyGit** [comment]
doesn't contain part 6 eventonica files, only the complete react_evetonica files

File: Assignments/React_Eventonica/react/src/components/generalFunc.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            .then((result) => filterOutEvents(eventId, result))
+}
+
+
+export {
+    getLocalStorage, 
+    setLocalStorage,
+    formatTime,
+    updateUserEvent,
+    deleteEventFromUser};
```
Comment (datadaveshin) [inline Assignments/React_Eventonica/react/src/components/generalFunc.js]: Call this `helperFunctions.js`? I thought it was a generally used component.

File: Assignments/React_Eventonica/react/src/Views/Ticketmaster.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            </div>
+            <br></br>
+            <div>
+                <h3>Save Event for User</h3>
+                <SaveTMEventToUsers sendData={this.storeUserResultChildValue.bind(this)}/>
+                <br></br>
+                <h4>User Events:</h4>
+                <ul id="user-saved-events">{this.state.userData}</ul>
+            </div>
+          </div>
```
Comment (datadaveshin) [inline Assignments/React_Eventonica/react/src/Views/Ticketmaster.js]: Could these somehow be a shared component?

**Comment by vegetabill** [comment]
@GrunuffyGit see comments above but I'm closing for now.

**Pull request closed without merging**